.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,10 @@ Remove this section if not relevant
 
 Yes/No
 
+### Commits signed with GPG?
+
+Yes/No
+
 Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.
+
+See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information signing commits with GPG.
```

Comment:
We have a section on our docs about this [here](https://docs.saltstack.com/en/latest/topics/development/contributing.html#gpg-verification). Should we link to those instead? I am not sure if that gains us anything, but it does have info on what the GPG bot is looking for.

Also, I think this sentence needs an `about` between `information` and `signing`.

Reply:
I thought about that but wanted to call it out here explicitly since it seems likely, at least for a while, that people will have questions and I wanted to make the path to an answer as short as possible from here.